app/src/main/java/com/cmput301w23t09/qrhunter/player/Player.java

```diff
@@ -39,13 +44,16 @@ public Player(UUID deviceId, String username, String phoneNo, String email) {
    * @param username username of the player
    * @param phoneNo phone number of the player
    * @param email email of the player
+   * @param qrCodeHashes qrcode hashes associated with player
    */
-  public Player(String documentId, UUID deviceId, String username, String phoneNo, String email) {
+  public Player(String documentId, UUID deviceId, String username, String phoneNo, String email, ArrayList<String> qrCodeHashes) {
```

Comment:
Can you rename qrCodeHashes to `scannedQRHashes` so it's more descriptive about what it represents?

Reply:
ok